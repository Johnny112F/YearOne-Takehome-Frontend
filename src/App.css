body {
  background: rgba(102, 68, 18, 0.975);
background: radial-gradient(circle, rgba(148, 33, 37, 0.526) 0%, rgba(54,48,53,1) 100%);
  height: fit-content;
  min-height: 100vh;
}
