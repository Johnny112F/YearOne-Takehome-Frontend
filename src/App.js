import logo from './logo.svg';
import './App.css';
import MoviesList from './MoviesList/MoviesList';

function App() {
  return (
    <div className="App">
      <MoviesList />
    </div>
  );
}

export default App;
