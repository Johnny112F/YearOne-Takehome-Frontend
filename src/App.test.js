import { render, screen } from '@testing-library/react';
import App from './App';


describe("snapshot tests for stability", function () {
  test("shows initial home search page", function () {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  })
  test('YearOne text appears twice on home page', () => {
    render(<App />);
    const yearOneAppearances = screen.queryAllByText("YearOne");
    expect(yearOneAppearances).toHaveLength(2);
  });
});