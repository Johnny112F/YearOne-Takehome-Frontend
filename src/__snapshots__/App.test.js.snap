// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot tests for stability shows initial home search page 1`] = `
<div>
  <div
    class="App"
  >
    <div
      class="NavBar"
    >
      <a
        href="/"
      >
        <span>
          YearOne
        </span>
        <span
          id="blue"
        >
          MovieRatings
        </span>
      </a>
    </div>
    <div
      class="MovieListPage"
    >
      <header
        class="MovieListPage-header"
      >
        <p
          class="MovieListPage-about"
        >
          An Application For Finding and Rating Films 
        </p>
        <h1>
          <span>
            YearOne
          </span>
          MovieRatings
        </h1>
        <p
          class="MovieListPage-credits"
        >
          API by TMDB
        </p>
      </header>
      <form
        class="SearchForm"
      >
        <input
          name="term"
          placeholder="Search movie titles.."
          value=""
        />
        <button
          type="Submit"
        >
          Search
        </button>
      </form>
      <section
        class="MovieListPage-pages-container"
      />
      <ul
        class="MoviesList"
      />
    </div>
    <section
      class="Contact"
    >
      <p>
        Questions or suggestions or both?
      </p>
      <p>
        Contact me 
        <a
          href="https://github.com/johnny112f"
          rel="noreferrer"
          target="_blank"
        >
          @johnny112f
        </a>
         on Github!
      </p>
    </section>
  </div>
</div>
`;
